README.md

```diff
@@ -41,20 +41,20 @@ manual steps in this section that will eventually be replaced with an installati
 
 All Selene applications are time zone agnostic.  It is recommended that the time zone on any server running Selene be UTC.
 
-## Postgres DB
-* Recommended server configuration: Ubuntu 18.04 LTS, 2 CPU, 4GB RAM, 50GB disk.
+### Postgres DB
+* Recommended server configuration: [Ubuntu 18.04 LTS (server install)](https://releases.ubuntu.com/bionic/), 2 CPU, 4GB RAM, 50GB disk.
 * Use the package management system to install Python 3.7, Python 3 pip and PostgreSQL 10  
 ```
 sudo apt-get install postgresql python3.7 python python3-pip
 ```
 * Set Postgres to start on boot  
-```
+``` 
 sudo systemctl enable postgresql
-```
+``` 
 * Clone the selene-backend and documentation repositories
 ```
 sudo mkdir -p /opt/selene
-sudo chown -R mycroft:users /opt/selene
+sudo chown -R $USER:users /opt/selene
```

Comment:
This is meant to be done as an application user.  Changing this command could be misleading as folks could make it be their own user.

Reply:
How about this extra note and a custom variable?

Reply:
Perhaps it's just because I setup my environment before going through the instructions. So I already had my users setup and none of them were mycroft. I don't think it's a critical thing though so happy to just switch back to the hardcoded mycroft user. If someone can't figure that out they really shouldn't be trying to run their own Selene instance.

Reply:
let's just leave it as mycroft user for now.

Reply:
Done